mne/viz/tests/test_topomap.py

```diff
@@ -182,6 +182,24 @@ def test_plot_topomap_animation(capsys):
     plt.close("all")
 
 
+def test_plot_topomap_animation_csd(capsys):
+    """Test topomap plotting of CSD data."""
+    from mne.preprocessing import compute_current_source_density
```

Comment:
you need to nest the import to avoid circular deps?

Reply:
No real reason why I import it here. I just looked at other tests in the file and noticed that they also import their dependencies inside e.g.: 
https://github.com/mne-tools/mne-python/blob/e572860cb1e806411834159cc2739c45ad326afd/mne/viz/tests/test_topomap.py#L758-L775

Happy to move it out of the test though.